### DISCUSSION TITLE
Maybe an error in the docs or something has changed with Ubuntu 20.04?

### DISCUSSION THREAD

**Discussion by stefan-tuellmann** [beeware/toga#1308] [Q&A]
From [https://github.com/beeware/toga#prerequisites](url)

Ubuntu 18.04 / Debian 10:
`sudo apt-get install python3-dev python3-gi python3-gi-cairo libgirepository1.0-dev libcairo2-dev libwebkitgtk-4.0-37 gir1.2-webkit2-4.0`

wasn't working for me. But i was able to install it with

`libwebkit2gtk-4.0-37`

I'm on Ubuntu 20.04. Maybe others will find this helpful, too?

Okay, now i found this in the tutorial:

`#Debian 10 has webkit2-4.0
 libwebkitgtk version seems very specific, but that is what it currently is @ 20190825
(venv) $ sudo apt-get update
(venv) $ sudo apt-get install python3-dev libgirepository1.0-dev libcairo2-dev libpango1.0-dev libwebkit2gtk-4.0-37 gir1.2-webkit2-4.0`

Seems to me a little bit strange.

**Comment by freakboy3742** [comment]
Thanks - looks like we've had some drift between the README and the tutorial. Nailing down system dependencies on Linux is really hard because (a) there are so many Linux distros; (b) the package names have a habit of changing over time; (c) once you've got your system working, you generally don't need to touch these system dependencies again; and (d) it's sometimes difficult to establish whether your system is working because of something that was installed by default because of your particular initial installation profile, or because of something you needed to install. 

Regardless - we definitely need to do a "clean room audit" of our install dependencies.
